src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -266,8 +267,13 @@ public LineGeometry3D Axes
 
         public PhongMaterial SelectedMaterial { get; set; }
 
-        public Transform3D Model1Transform { get; private set; }
-
+        public Transform3D StaticTransform
```

Comment:
`StaticTransform` is an overload of the term `static` in the language. Can we call this `BaseTransform` or something?